client/css/common.css

```diff
@@ -0,0 +1,89 @@
+/* Overwriting Bootstrap CSS */
+.nav-link {
+    color: #333;
+}
+
+.nav-link.active {
+    color: #007bff;
+}
+
+.row {
+    margin-left: 0px !important;
+    margin-right: 0px !important;
+}
+
+/* Top bar */
+#top-bar {
+    padding-bottom: 0px;
+    padding-top: 0px;
+}
+
+/* Sidebar */
+#sidebar {
+    width: inherit;
+    min-width: 220px;
+    max-width: 220px;
+    background-color: #f5f5f5;
+    opacity: 1;
+    float: left;
+    height: 100%;
+    position: relative;
+    overflow-y: auto;
+    overflow-x: hidden;
+    z-index: 10; /* in front of overlay */
```

Comment:
I would try to steer away from `z-index` (it becomes a hassle later on if all ur elements have a z-index). Is there a reason why you have this? Or is this the *only* way we can put the element in front of the overlay?